Livecheckables/freetds.rb

```diff
@@ -1,4 +1,4 @@
 class Freetds
-  livecheck :url   => "https://fossies.org/linux/privat/",
-            :regex => /HREF="freetds-([\d.]+)\.tar/
+  livecheck :url   => "http://www.freetds.org/files/stable/",
+            :regex => /freetds-([\d.]+)\.tar\.gz/
```

Comment:
```suggestion
            :regex => /href=.+?freetds-v?(\d+(?:\.\d+)+)\.t/i
```

There are a few things that can be improved here, while we're editing this livecheckable:

* We almost always want to search within the `href` attribute on archive index pages like these, so removing that from the regex here isn't appropriate. We can just make it lowercase and/or make the entire regex case insensitive.
* We can update the part that matches the numeric version `([\d.]+)` to the more explicit regex that we use for this (`v?(\d+(?:\.\d+)+)`), which only matches what we want (and wouldn't match something weird like `.....`).
* There's nothing to say that the archive format couldn't become `tgz` or `tar.xz` in the future, which wouldn't work with either the old or the new regex in this PR. We just use `\.t` to match extensions like `.tgz`, `.tar.*`, etc, so there's less of a chance of it breaking in the future.